src/templates/subscribe.html

```diff
@@ -0,0 +1,30 @@
+{% extends "base.html" %}
+{% block content %}
+<div class="container">
+    <h1>Get updated</h1>
+    <p>Get the newst Festival news right into your inbox, for free. As soon as something changes, you will be notfied.</p>
+    <div class="row">
+        <div class="col-5">
+
+            <form action="{{ url_for('success') }}" method="post">
+              <div class="form-group">
+                <label for="exampleInputEmail1">Email address*</label>
```

Comment:
Old code from copy-paste? the value of "for" should be the id of the input element that this label is linked to. Same for the other two labels.